context_test.go

```diff
@@ -475,6 +485,7 @@ func TestDetectGOPATH(t *testing.T) {
 	}
 
 	for _, tc := range testcases {
+		th.TempDir(filepath.Join("go", "src/github.com/username/package"))
```

Comment:
This is no longer needed. It's a duplication of https://github.com/golang/dep/pull/764/files#diff-bf503e77c0e6429882a1ec6525e4cc95R473